### PR TITLE
Bug/cuke or profileupdations

### PR DESCRIPTION
**Pull request by Soujanya2107**

### CODE REVIEW

**Comment by Soujanya2107** [comment]
Password Update:
Changes the password, checks for the response
Then logs out and log in with the new password , thus verifies the updation

Email Update:
Changes the Email ,checks for the success message , refreshes and logsOut
Confirms that a mail is received in the mail server inbox, verifies the link and confirms account, and then logs in with updated email address

Phone number updation:
Changes the phone number, checked for success message and page is refreshed and checks for updated mobile number

File: tests/javascript-cucumber/Features/step_definitions/steps.js [hunk 1]
```diff
@@ -254,12 +255,33 @@ module.exports = function () {
     });
 
     this.When(/^user logs in to their validated account$/, async function () {
+        await signup(this.driver, randomPrefix);
```
Comment (radek1st) [inline tests/javascript-cucumber/Features/step_definitions/steps.js:258]: here and below you repeat a lot of the same code

File: tests/javascript-cucumber/Features/profile_update.feature [hunk 2]
```diff
@@ -1,25 +1,30 @@
...
     When user fills in the new password and confirms with the existing password
     And user clicks on change password button
-    Then user is displayed with success message
+    And user is displayed with password updated message 
+    Then user logs in with new password
 
   Scenario: User updates email
 
-    Given user logs in to their validated account
+    Given user logs in to their validated account to change email
```
Comment (radek1st) [inline tests/javascript-cucumber/Features/profile_update.feature:14]: why do you need to add a new step?

**Comment by radek1st** [comment]
Last thing to update before we can merge this in: 

* please add the before hook to generate random email and 
* remove code duplication for `Given user logs in to their validated account` step

**Comment by Soujanya2107** [comment]
> Last thing to update before we can merge this in:
> 
> * please add the before hook to generate random email and
> * remove code duplication for `Given user logs in to their validated account` step

--removed the redundant "Given user logs in to their validated account"
--Added global function to randomise prefix in world.js

**Pull request merged**